### ISSUE TITLE
[Bug] markLine label vertical distance didn't work?

### ISSUE THREAD

**Issue by CrazyRedTime** [apache/echarts#19092]
### Version

5.4.1 - 5.4.3

### Link to Minimal Reproduction

https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=PYBwLglsB2AEC8sDeAoWsAeBBDEDOAXMmurGAJ4gCmRA5AMYCGYVA5sAE7m0A0J6AE2aMiAbVoBZGL1i0AKgFcqM2gHUqAlXIAWClQDEOEFQGVmphdFoBdEgF8-6cjnxFUpMpRqyAbowA2SrT2jrB4VEZUhLCi_MQegsJiAIwArAAMPLAATADMmTnZACxZ2ckAHFnJualVRQDspQBs6dahHhTUdP4Q0MrtpAC2jBwA1gAyvd7uCYlgIjFxs-gzywnQjIPetNAK_v68S2uwzrjRydmZR2v-jABGVP5u18cAZpzDYCwcdEh3dodjssQMA8BBIDA6Hh5hwwICgaQHC9Zj0-iYKP5psjlvRgP5OHQOBp4QjSAB3CACMDaIjJAZApGkxkI1YI06uWAXK6k9AbLZ0KjQASC4VCkkI24PJ7xHnoEFgiHQAVi-kI94cT7fX7_AAUSHodgAlOLSRSqTTYEVubLYMAfBFXviyXQ7kTGKMsPsTcdmaTUVR0eRMc8bbBcfifrIiZpVUCzdTabHlr6fUmPKygXzttCRnC0wl2dFsi18x5JY8Qzb1ZqItqAaWEvLwVAlbIc7DvUC8NpgM6yBwlA2PAJ8PNoPRvKJLlV0q0h7AU2tF8sM8dC0Ri9bSVnlQI8IwBLARYw8Ifj6eT53luXpaugU3Fbur2tq8wtbI_vXsbM7Q6nS63Q9L153QeMLStEDYBHHNx0nadYBaNpv0Red_UDYMZVlcMCSjYl52XWY7BeWxCLiIj0FsOwAG4gA

### Steps to Reproduce

1. Create mark line
2. Set label => distance with array(as it says in [docs](https://echarts.apache.org/en/option.html#series-line.markLine.label.distance), as I did in the example on the 48 and 59 strings
3. Profit

### Current Behavior

Second value in array just ignored

### Expected Behavior

Second value in array must be set vertical distance of the markLine's label

### Environment

```markdown
- OS: windows 10
- Browser: chrome, opera
- Framework: vanilla js
```

**Comment by helgasoft** [comment]
Could be considered a documentation bug.
Vertical (second) _distance_ works only for [markLine.label.position](https://echarts.apache.org/en/option.html#series-line.markLine.label.position) set to '**middle**', but not for 'start' or 'end'.

**Comment by CrazyRedTime** [comment]
> Could be considered a documentation bug. Vertical (second) _distance_ works only for [markLine.label.position](https://echarts.apache.org/en/option.html#series-line.markLine.label.position) set to '**middle**', but not for 'start' or 'end'.

Padding top in label is also documentation bug?

**Comment by helgasoft** [comment]
yes, like _distance_, **padding** works only at _position:'middle'_ -  [Demo Code](https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=PYBwLglsB2AEC8sDeAoWsAeBBDEDOAXMmurGAJ4gCmRA5AMYCGYVA5sAE7m0A0J6AE2aMiAbVoBZGL1i0AKgFcqM2gHUqAlXIAWClQDEOEFQGVmphdFoBdEgF8-6cjnxFUpMpRqyAbowA2SrT2jrB4VEZUhLCi_MQegsJiAIwArAAMPLAATADMmTnZACxZ2ckAHFnJualVRQDspQBs6dahHhTUdP4Q0MrtpAC2jBwA1gAyvd7uCYlgIjFxszOzHtCMg960ePMcYLxLq8640dktA6uw_owARlT-boeXAGacw2AsHHRIN3YHl7MQMA8BBIDA6DsRvsLgC8NpgAB3IhgDhKGGXAT4ebQejeUTZArJdKtJ4JOyk9AOCnxAFOFync7U9DrTZ0KjQAR4RgCWDsoR4Hl8xgC4X_WlXW73R7i0hAkFg6B0QYQAQCfz9JmkV4cd6fb6_MUy4A-CLPfyIug3DhURijLD-fyG8UIlVgbREIqZTWCLGMHF4glZT1tb2wEDczHQVhibK1WBpLIFVro1Y3TgCCKqV3u-Mp2ZpjgZjgAYWA5q-smtmk1VJlPT6Jgo6ulMtg9DLnDoVeCMvJtL7q1sswHlJItjsAG4gA)
